_redirects

```diff
@@ -1,50 +1,37 @@
-https://idrc.ocad.ca/*                                                                  https://idrc.ocadu.ca/:splat                                            301!
-https://www.idrc.ocad.ca/*                                                              https://idrc.ocadu.ca/:splat                                            301!
```

Comment:
Have the redirects like this been abandoned or handled elsewhere?

Reply:
As you may remember, Cloudflare doesn't handle domain-level redirects. These are now handled in our redirects service on the Docker host.